sem/manager.py

```diff
@@ -389,10 +459,20 @@ def get_missing_simulations(self, param_list, runs=None,
             runs (int): an integer representing how many repetitions are wanted
                 for each parameter combination, None if the dictionaries in
                 param_list already feature the desired RngRun value.
+            log_component (dict): This dictionary contains {logComponent:LogLevel}
+                (to enable) as {key:value} pairs. Passing this parameter specifies 
+                the function to only look in the results with the logging flag enabled 
+                (This ensures that the results of simulations ran without run_with_log
+                API are left untouched).
         """
 
         params_to_simulate = []
 
+        # For simulations where logging is enabled it does not make sense to run
+        # simulations more than one time
+        if log_component is not None:
+            runs=1
```

Comment:
it does make sense: different seeds return different results and possibly different errors

Reply:
Agree! Plus, I expect that users will want to replicate a specific value for the `RngRun` parameter: in this case, runs will probably be `None`, and the `RngRun` will be specified in the parameter combinations inside `param_list`